src/dialogs/more-info/controls/more-info-camera.ts

```diff
@@ -76,7 +74,7 @@ class MoreInfoCamera extends UpdatingElement {
           this.stateObj.entity_id
         );
 
-        if (Hls) {
+        if (Hls.isSupported()) {
```

Comment:
This is weird. Does hls better check for supported than native? We can check the return type of can play too

Reply:
So HLS.js just have to check if MediaSourceExtension is supported, which is an easy check on the window. I have no clue why canPlayType returns an empty string, maybe, or probably.  All 3 of them sound pretty uncertain. All the demos I’ve looked at use HLS.js first and fallback to canPlayType, then unsupported.

Reply:
![image](https://user-images.githubusercontent.com/1444314/54484987-64e5fb80-482e-11e9-894f-7b622d45f747.png)

https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/canPlayType